<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigureLayout</class>
 <widget class="QWidget" name="ConfigureLayout">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>950</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>480</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>422</width>
        <height>932</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="layout_group">
         <property name="title">
          <string>Screens</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QWidget" name="widget_layout" native="true">
            <layout class="QHBoxLayout" name="screen_layout_group">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="layout_label">
               <property name="text">
                <string>Screen Layout:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="layout_combobox">
               <item>
                <property name="text">
                 <string>Default</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Single Screen</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Large Screen</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Side by Side</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Separate Windows</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Custom Layout</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="toggle_swap_screen">
            <property name="text">
             <string>Swap Screens</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="toggle_upright_screen">
            <property name="text">
             <string>Rotate Screens Upright</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget" native="true">
            <layout class="QHBoxLayout" name="proportion_layout">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Large Screen Proportion:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="large_screen_proportion">
               <property name="minimum">
                <double>1.000000000000000</double>
               </property>
               <property name="maximum">
                <double>16.000000000000000</double>
               </property>
               <property name="value">
                <double>4.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="bg_color_group" native="true">
            <layout class="QHBoxLayout" name="bg_color_group_2">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="bg_label">
               <property name="text">
                <string>Background Color:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="bg_button">
               <property name="maximumSize">
                <size>
                 <width>40</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="custom_layout_group">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="title">
             <string>Custom Layout</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QGroupBox" name="gb_top_screen">
                 <property name="title">
                  <string>Top Screen</string>
                 </property>
                 <layout class="QGridLayout" name="gridLayout_2">
                  <item row="0" column="0">
                   <widget class="QLabel" name="lb_top_left">
                    <property name="text">
                     <string>Left</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QSpinBox" name="custom_top_left">
                    <property name="buttonSymbols">
                     <enum>QAbstractSpinBox::NoButtons</enum>
                    </property>
                    <property name="maximum">
                     <number>2147483647</number>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <widget class="QLabel" name="lb_top_top">
                    <property name="text">
                     <string>Top</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="1">
                   <widget class="QSpinBox" name="custom_top_top">
                    <property name="buttonSymbols">
                     <enum>QAbstractSpinBox::NoButtons</enum>
                    </property>
                    <property name="maximum">
                     <number>2147483647</number>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="0">
                   <widget class="QLabel" name="lb_top_right">
                    <property name="text">
                     <string>Right</string>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="1">
                   <widget class="QSpinBox" name="custom_top_right">
                    <property name="buttonSymbols">
                     <enum>QAbstractSpinBox::NoButtons</enum>
                    </property>
                    <property name="maximum">
                     <number>2147483647</number>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="0">
                   <widget class="QLabel" name="lb_top_bottom">
                    <property name="text">
                     <string>Bottom</string>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="1">
                   <widget class="QSpinBox" name="custom_top_bottom">
                    <property name="buttonSymbols">
                     <enum>QAbstractSpinBox::NoButtons</enum>
                    </property>
                    <property name="maximum">
                     <number>2147483647</number>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QGroupBox" name="gb_bottom_screen">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="title">
                  <string>Bottom Screen</string>
                 </property>
                 <layout class="QGridLayout" name="gridLayout">
                  <item row="0" column="0">
                   <widget class="QLabel" name="lb_bottom_left">
                    <property name="text">
                     <string>Left</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QSpinBox" name="custom_bottom_left">
                    <property name="buttonSymbols">
                     <enum>QAbstractSpinBox::NoButtons</enum>
                    </property>
                    <property name="maximum">
                     <number>2147483647</number>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <widget class="QLabel" name="lb_bottom_top">
                    <property name="text">
                     <string>Top</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="1">
                   <widget class="QSpinBox" name="custom_bottom_top">
                    <property name="buttonSymbols">
                     <enum>QAbstractSpinBox::NoButtons</enum>
                    </property>
                    <property name="maximum">
                     <number>2147483647</number>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="0">
                   <widget class="QLabel" name="lb_bottom_right">
                    <property name="text">
                     <string>Right</string>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="1">
                   <widget class="QSpinBox" name="custom_bottom_right">
                    <property name="buttonSymbols">
                     <enum>QAbstractSpinBox::NoButtons</enum>
                    </property>
                    <property name="maximum">
                     <number>2147483647</number>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="0">
                   <widget class="QLabel" name="lb_bottom_bottom">
                    <property name="text">
                     <string>Bottom</string>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="1">
                   <widget class="QSpinBox" name="custom_bottom_bottom">
                    <property name="buttonSymbols">
                     <enum>QAbstractSpinBox::NoButtons</enum>
                    </property>
                    <property name="maximum">
                     <number>2147483647</number>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLabel" name="lb_opacity_second_layer">
                 <property name="text">
                  <string>Bottom Screen Opacity:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="custom_second_layer_opacity">
                 <property name="buttonSymbols">
                  <enum>QAbstractSpinBox::PlusMinus</enum>
                 </property>
                 <property name="minimum">
                  <number>10</number>
                 </property>
                 <property name="maximum">
                  <number>100</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>layout_combobox</tabstop>
  <tabstop>toggle_swap_screen</tabstop>
  <tabstop>toggle_upright_screen</tabstop>
  <tabstop>large_screen_proportion</tabstop>
  <tabstop>bg_button</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
